### ISSUE TITLE
Options not working babel-plugin-transform-inline-environment-variables

### ISSUE THREAD

**Issue by josepbordesjove** [babel/minify#784]
Hello,
I've been trying to use a new inline variable to run my react-native app in multiple environments, and I didn't figure it out how to use the var "ENVIRONMENT", and not to destroy NODE_ENV. The code I tried is the one below, we are using react: 16.2.0 and react-native: 0.52.2. Thank you for your time, is really appreciated.

```javascript
{
  "presets": ["react-native"]
  "plugins": [["transform-inline-environment-variables", {
      "exclude": ["NODE_ENV"],
      "include": ["ENVIRONMENT"],
    }]
  ]
}
```

**Comment by vigneshshanmugam** [comment]
Fixed in [0.3.0](https://github.com/babel/minify/releases/tag/gulp-babel-minify%400.3.0)

**Comment by josepbordesjove** [comment]
@vigneshshanmugam Thank you :D
